### ISSUE TITLE
Experiment - Different optimizer methods with to goal of reducing training time

### ISSUE THREAD

**Issue by rylandtikes** [rylandtikes/cs598-project#26]
Different optimizer methods with to goal of
reducing training time, as it is considerable on
the hardware available to us.

**Comment by s-enright** [comment]
AdamW replaced Adam, to separate regularization from weight decay. No performance impact.
